sdk/evaluation-context.json

```diff
@@ -53,12 +123,121 @@
             ],
             "title": "IdentityContext",
             "type": "object"
+        },
+        "SegmentContext": {
+            "description": "Represents a segment context for feature flag evaluation.",
+            "properties": {
+                "key": {
+                    "description": "Key used for % split segmentation.",
+                    "title": "Key",
+                    "type": "string"
+                },
+                "name": {
+                    "description": "The name of the segment.",
+                    "title": "Name",
+                    "type": "string"
+                },
+                "rules": {
+                    "items": {
+                        "$ref": "#/$defs/SegmentRule"
+                    },
+                    "title": "Rules",
+                    "type": "array"
+                }
+            },
+            "required": [
+                "key",
```

Comment:
Again, it looks odd that `key` is required given not every segment uses `%` rules.